ext/rbs_extension/parser.c

```diff
@@ -52,6 +52,11 @@ typedef struct {
   VALUE rest_keywords;
 } method_params;
 
+typedef struct {
```

Comment:
Looks like the struct is no longer used.

Reply:
thx, removed.